api/proto/banyandb/model/v1/query.proto

```diff
@@ -68,8 +72,21 @@ message Condition {
 
 // tag_families are indexed.
 message Criteria {
-    string tag_family_name = 1;
-    repeated model.v1.Condition conditions = 2;
+    repeated Expression exps = 1;
```

Comment:
What does the repeated imply here? And or OR?

Reply:
I want to recheck here too. If Expression is planned to support nested, then Expression should be a hierarchy structure, rather than a `repeated`. This seems an intermediate status

Reply:
Add inner expression here

Reply:
Once the nested expression mechanism is added, we don't need repeatable exps here. Eventually, we need one exp at the top level.

Reply:
@hanahmily I think this has not been 100% resolved, the `repeated` is still arguable.

Reply:
Push down "repeated" fields, that could support nested operations.